src/Kdyby/Translation/DI/TranslationExtension.php

```diff
@@ -207,12 +245,39 @@ protected function loadConsole(array $config)
 	protected function loadDumpers()
 	{
 		$builder = $this->getContainerBuilder();
+		$config = $this->getConfig();
 
 		foreach ($this->loadFromFile(__DIR__ . '/config/dumpers.neon') as $format => $class) {
 			$builder->addDefinition($this->prefix('dumper.' . $format))
 				->setClass($class)
 				->addTag(self::DUMPER_TAG, $format);
 		}
+
+		$dumper = $config['database']['dumper'];
+		$isDumper = false;
+		if ($dumper !== NULL) {
+			$isDumper = true;
+		} else {
+			$loader = $config['database']['loader'];
+			if (in_array($loader, array_keys(self::$dbLoaders))) { //if you register doctrine of nettedb loader, dumper is also registered
+				$isDumper = true;
+				$dumper = $loader;
+			}
+		}
+		if ($isDumper) {
+			if (in_array($dumper, array_keys(self::$dbDumpers))) {
+				$class = self::$dbDumpers[$dumper];
+			} else {
+				$class = $dumper;
+			}
+			list($class) = self::filterArgs($class);
```

Comment:
asi bude lepsi pouzit `Compiler::parseService`, pak muze uzivatel lepe specifikovat argumenty, setup apod. to same u loaderu